resources/assets/js/components/people/activity/ActivityList.vue

```diff
@@ -0,0 +1,196 @@
+<style scoped>
+</style>
+
+<template>
+  <div>
+    <div class="">
+      <h3 class="mb2">
+        🍿 {{ $t('people.activity_title') }}
+
+        <span class="fr relative" style="top: -7px;">
+          <a v-if="displayLogActivity == false" class="btn edit-information" @click="displayLogActivity = true">
+            {{ $t('people.activities_add_activity') }}
+          </a>
+          <a v-if="displayLogActivity" class="btn edit-information" @click="displayLogActivity = false">
+            {{ $t('app.cancel') }}
+          </a>
+        </span>
+      </h3>
+    </div>
+
+    <!-- BLANK STATE -->
+    <div v-if="!displayLogActivity && activities.length == 0" class="w-100">
+      <div class="bg-near-white tc pa3 br2 ba b--light-gray">
+        <p>{{ $t('people.activities_blank_title', { name: name }) }}</p>
+        <a class="pointer" @click.prevent="displayLogActivity = true">
+          {{ $t('people.activities_blank_add_activity') }}
+        </a>
+      </div>
+    </div>
+
+    <!-- LOG AN ACTIVITY -->
+    <create-activity
+      :hash="hash"
+      :name="name"
+      :display-log-activity="displayLogActivity"
+      @update="updateList($event)"
+      @cancel="displayLogActivity = false"
+    />
+
+    <!-- LIST OF ACTIVITIES -->
+    <div v-for="activity in activities" :key="activity.id" class="ba br2 b--black-10 br--top w-100 mb2">
+      <h2 class="pl2 pr2 pt3 f5">
+        {{ activity.summary }}
+      </h2>
+
+      <div v-if="activity.description" class="pl2 pr2 pb3" v-html="activity.description">
```

Comment:
`v-html` can lead to XSS vulnerability, are you sure you want to use it?

Reply:
@asbiin description can contain markdown which needs to be rendered. There is no other way to display Markdown then.... OR I have to parse markdown in Vue and not in the backend. What do you think?

Reply:
Oh well, is description is parsed on the backend, it should be OK.
Anyways, I'm not sure, but we could fix this (it's used in other part of the application) later.